### ISSUE TITLE
love-release does not make .love file

### ISSUE THREAD

**Issue by HDictus** [MisterDA/love-release#65]
I'm running lua 5.1 on ubuntu 18.04 the conf.lua in question is
```lua
function love.conf(t)
  t.releases = {
     title = 'particles',
     package = '-',
     loveVersion = '11.2',
     version = nil,
     author = nil,
     email = nil,
     description = nil,
     homepage = nil,
     identifier = nil,
     excludeFileList = { ''} ,
     compile = false,
     projectDirectory = '<dir>/particles/.',
     releaseDirectory = '<dir>/particles/release',
  }
end
```

In `<dir>`, I run `love-release particles/release particles -W` and I get 
```
/usr/bin/lua5.1: ...local/share/lua/5.1/love-release/scripts/windows.lua:46: particles.love: No such file or directory
stack traceback:
	[C]: in function 'assert'
	...local/share/lua/5.1/love-release/scripts/windows.lua:46: in function 'release'
	...local/share/lua/5.1/love-release/scripts/windows.lua:82: in function <...local/share/lua/5.1/love-release/scripts/windows.lua:77>
	(tail call): ?
	/usr/local/share/lua/5.1/love-release/pipes/args.lua:108: in function 'args'
	...cks/rocks-5.1/love-release/2.0.12-2/bin/love-release:11: in main chunk
	[C]: ?
```
And indeed there is no particles.love anywhere.
If I create a .love file myself and put it in particles/release it runs to completion
`script:createLoveFile` is called, but somehow the result isn't written to file.

**Comment by HDictus** [comment]
I've been having a look at it, and it seems that there isn't anything in project:fileList(). All there is to add for this project is conf.lua and main.lua

**Comment by HDictus** [comment]
By default, it seems that the string '' is added to excludeFileList, which matches every string for indices 0, 1. This was resulting in every file being excluded.
I'm thinking it'd be best to ensure '' exclude no file in isExcluded

**Comment by MisterDA** [comment]
Fixed by PR #66 and commit 5be593f.
